diff --git a/main.go b/main.go
index d5da5b0..8436963 100644
--- a/main.go
+++ b/main.go
@@ -21,10 +21,12 @@ import (
 	"gitlab.com/lightmeter/controlcenter/logeater"
 	"gitlab.com/lightmeter/controlcenter/logeater/dirwatcher"
 	"gitlab.com/lightmeter/controlcenter/po"
-	"gitlab.com/lightmeter/controlcenter/staticdata"
 	"gitlab.com/lightmeter/controlcenter/util"
 	"gitlab.com/lightmeter/controlcenter/version"
+
+	"gitlab.com/lightmeter/controlcenter/staticdata"
 	"gitlab.com/lightmeter/controlcenter/workspace"
+	_ "github.com/shurcooL/vfsgen"
 )
 
 type watchableFilenames []string
